README.md

```diff
@@ -132,33 +121,56 @@ Local torch tensor on rank  0 :  tensor([0, 1, 2, 3, 4], dtype=torch.int32)
+scientific and data science applications as effortless as using NumPy.
 
-* [mpi4py docs](https://mpi4py.readthedocs.io/en/stable/tutorial.html)
-* [Tutorial](https://www.kth.se/blogs/pdc/2019/08/parallel-programming-in-python-mpi4py-part-1/)
+Heat allows you to tackle your actual Big Data challenges that go beyond the
+computational and memory needs of your laptop and desktop.
+ -->
+
+# Support Channels
+
+We use [GitHub Discussions](https://github.com/helmholtz-analytics/heat/discussions) as a forum for questions about Heat.
+If you found a bug or miss a feature, then please file a new [issue](https://github.com/helmholtz-analytics/heat/issues/new/choose).
 
-# Contribution guidelines
 
-**We welcome contributions from the community, if you want to contribute to Heat, be sure to review the [Contribution Guidelines](contributing.md) before getting started!**
+# Contribution guidelines
 
-We use [GitHub issues](https://github.com/helmholtz-analytics/heat/issues) for tracking requests and bugs, please see [Discussions](https://github.com/helmholtz-analytics/heat/discussions) for general questions and discussion, and You can also get in touch with us on [Mattermost](https://mattermost.hzdr.de/signup_user_complete/?id=3sixwk9okpbzpjyfrhen5jpqfo). You can sign up with your GitHub credentials. Once you log in, you can introduce yourself on the `Town Square` channel.
+**We welcome contributions from the community, if you want to contribute to Heat, be sure to review the [Contribution Guidelines](contributing.md) and [Resources for MPI programming](#resources-for-mpi-programming)  before getting started!**
 
-Small improvements or fixes are always appreciated; issues labeled as **"good first issue"** may be a good starting point.
+We use [GitHub issues](https://github.com/helmholtz-analytics/heat/issues) for tracking requests and bugs, please see [Discussions](https://github.com/helmholtz-analytics/heat/discussions) for general questions and discussion. You can also get in touch with us on [Mattermost](https://mattermost.hzdr.de/signup_user_complete/?id=3sixwk9okpbzpjyfrhen5jpqfo) (sign up with your GitHub credentials). Once you log in, you can introduce yourself on the `Town Square` channel.
 
 If you’re unsure where to start or how your skills fit in, reach out! You can ask us here on GitHub, by leaving a comment on a relevant issue that is already open.
 
 **If you are new to contributing to open source, [this guide](https://opensource.guide/how-to-contribute/) helps explain why, what, and how to get involved.**
 
+
+## Resources for MPI programming
+
+* [Heat Tutorials](https://heat.readthedocs.io/en/latest/tutorials.html)
+* [Heat API Reference](https://heat.readthedocs.io/en/latest/autoapi/index.html)
+
+### Parallel Computing and MPI:
+
+* @davidhenty's [course](https://www.archer2.ac.uk/training/courses/200514-mpi/)
+* Wes Kendall's [Tutorials](https://mpitutorial.com/tutorials/)
+
```

Comment:
maybe we can add R. Rabenseifners course material: 
* Rolf Rabenseifners [MPI course material](https://www.hlrs.de/training/self-study-materials/mpi-course-material) (including C, Fortran **and** Python via `mpi4py`)